packages/oui-checkbox/src/checkbox.html

```diff
@@ -1,28 +1,18 @@
-<div class="oui-checkbox" data-ng-class="{
-    'oui-checkbox_big': $ctrl.big,
-    'oui-checkbox_thumbnail': $ctrl.thumbnail
-}">
+<div class="oui-checkbox">
   <input class="oui-checkbox__input"
          type="checkbox"
-         id="{{:: $ctrl.name }}"
+         id="{{:: $ctrl.id }}"
          name="{{:: $ctrl.name }}"
-         data-ng-model="$ctrl.value"
-         data-ng-change="$ctrl.callOnChange($ctrl.value)"
-         data-ng-checked="$ctrl.checked"
+         data-ng-model="$ctrl.model"
+         data-ng-change="$ctrl.onChange({ model: $ctrl.model })"
```

Comment:
@frenautvh didn't make the same thing in his PR. We should discuss about the format of our events.

Reply:
Changed to `modelValue`